stubs/inertia/resources/js/Pages/Profile/ConnectedAccountsForm.vue

```diff
@@ -0,0 +1,144 @@
+
+                <template #footer>
+                    <jet-secondary-button @click.native="confirmingRemove = false">
+                        Nevermind
+                    </jet-secondary-button>
+
+                    <jet-button class="ml-2"
+                                @click.native="removeConnectedAccount(accountId)"
+                                :class="{ 'opacity-25': form.processing }" :disabled="form.processing">
+                        Remove Connected Account
+                    </jet-button>
+                </template>
+            </jet-dialog-modal>
+        </template>
+    </jet-action-section>
+</template>
+
+<script>
+    import JetActionLink from '@/Jetstream/ActionLink';
+    import JetActionMessage from '@/Jetstream/ActionMessage';
+    import JetActionSection from '@/Jetstream/ActionSection';
+    import JetButton from '@/Jetstream/Button';
+    import JetConnectedAccount from '@/Jetstream/ConnectedAccount';
+    import JetDialogModal from '@/Jetstream/DialogModal';
+    import JetInput from '@/Jetstream/Input';
+    import JetInputError from '@/Jetstream/InputError';
+    import JetSecondaryButton from '@/Jetstream/SecondaryButton';
+
+    export default {
+        props: ['accounts', 'providers', 'hasPassword'],
+
+        components: {
+            JetActionLink,
+            JetActionMessage,
+            JetActionSection,
+            JetButton,
+            JetConnectedAccount,
+            JetDialogModal,
+            JetInput,
+            JetInputError,
+            JetSecondaryButton,
+        },
+
+        data() {
+            return {
+                confirmingRemove: false,
+                accountId: null,
+
+                form: this.$inertia.form({
+                    '_method': 'DELETE',
+                }, {
+                    bag: 'removeConnectedAccount'
+                })
+            };
+        },
+
+        methods: {
+            confirmRemove(id) {
+                this.form.password = '';
+
+                this.accountId = id;
+
+                this.confirmingRemove = true;
+
+                setTimeout(() => {
+                    this.$refs.password.focus();
+                }, 250);
+            },
+
+            hasAccountForProvider(provider) {
+                return this.accounts.filter(account => account.provider_name === provider).length > 0;
+            },
+
+            getAccountForProvider(provider) {
+                if (this.hasAccountForProvider(provider)) {
+                    return this.accounts.filter(account => account.provider_name === provider).shift();
+                }
+
+                return null;
+            },
+
+            removeConnectedAccount(id) {
+                this.form.post(`/user/connected-account/${ id }`, {
+                    preserveScroll: true
+                }).then(response => {
+                    if (!this.form.hasErrors()) {
+                        this.confirmingRemove = false;
+                    }
```

Comment:
https://inertiajs.com/manual-visits#promise-deprecation

```suggestion
                    onSuccess: () => {
                        if (! this.form.hasErrors()) {
                            this.confirmingRemove = false;
                        }
                    },
```